logviewer/src/main/java/org/togetherjava/logwatcher/views/MainLayout.java

```diff
@@ -0,0 +1,172 @@
+package org.togetherjava.logwatcher.views;
+
+import com.google.common.collect.Sets;
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.applayout.AppLayout;
+import com.vaadin.flow.component.applayout.DrawerToggle;
+import com.vaadin.flow.component.avatar.Avatar;
+import com.vaadin.flow.component.button.ButtonVariant;
+import com.vaadin.flow.component.contextmenu.ContextMenu;
+import com.vaadin.flow.component.html.*;
+import com.vaadin.flow.router.PageTitle;
+import com.vaadin.flow.router.RouterLink;
+import org.slf4j.LoggerFactory;
+import org.togetherjava.logwatcher.accesscontrol.AllowedRoles;
+import org.togetherjava.logwatcher.accesscontrol.Role;
+import org.togetherjava.logwatcher.users.AuthenticatedUser;
+import org.togetherjava.logwatcher.views.logs.LogsView;
+import org.togetherjava.logwatcher.views.logs.StreamedView;
+import org.togetherjava.logwatcher.views.usermanagement.UserManagement;
+import org.togetherjava.tjbot.db.generated.tables.pojos.Users;
+
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Stream;
+
+/**
+ * The main view is a top-level placeholder for other views.
+ */
+@PageTitle("Main")
+@SuppressWarnings({"java:S1192"})
```

Comment:
Whats this warning, why did u suppress it?

Reply:
That's the Comment below it